assets/react/v3/shared/components/Container.tsx

```diff
@@ -0,0 +1,18 @@
+import { css } from "@emotion/react";
+import { ReactNode } from "react";
+
+const CONTAINER_WIDTH = 1116;
```

Comment:
could we implement something which will change according to the active device breakpoint?

Reply:
Not yet think about responsiveness.